android/app/src/main/java/com/hotwirerndemo/RNVisitableView.kt

```diff
@@ -60,91 +79,63 @@ class RNVisitableView (context: Context) : LinearLayout(context), TurboSessionCa
         super.onLayout(changed, l, t, r, b)
         val width = r - l
         val height = b - t
-        session.webView.layout(0, 0, width, height)
+        session.session.webView.layout(0, 0, width, height)
```

Comment:
Minor detail but `session.session` reads pretty bad. You could consider renaming the `session` property in `RNSession` to eg. `turboSession` to be explicit about the session type.

Reply:
👀 It will be fixed in the following PR